assets/src/components/block-navigation/edit.css

```diff
@@ -190,3 +190,36 @@
 		width: 60px;
 	}
 }
+
+/**
+ * Drag & Drop support
+ */
+
+.block-editor-block-navigation__list .block-editor-block-navigation__item {
+	position: relative;
+}
```

Comment:
Wondering if we should also change the pointer to match with dragging the blocks for example. Thoughts?

Reply:
True. Good point!

Reply:
Fixed now.